Source/CourseOutlineHeaderView.swift

```diff
@@ -77,26 +76,32 @@ public class CourseOutlineHeaderView: UIView {
         }
         
         viewButton.snp.makeConstraints { make in
-            make.trailing.equalTo(self.snp.trailing).offset(-StandardHorizontalMargin)
-            make.centerY.equalTo(self)
-            make.top.equalTo(self).offset(5)
-            make.bottom.equalTo(self).offset(-5)
+            make.trailing.equalTo(self.snp.trailing)
```

Comment:
Following will do the work

```suggestion
            make.trailing.equalTo(self)
```